packages/crypto/README.md

```diff
@@ -0,0 +1,3 @@
+# @turnkey/crypto
+
+This package consolidates some common cryptographic utilities used across our applications, particularly primitives related to keys, encoding/decoding and encryption.
```

Comment:
`keys, encryption, and decryption` (assuming we relegate all encoding/decoding stuff to `tkhq/encoding`)